### ISSUE TITLE
Breaking on function with docstring

### ISSUE THREAD

**Issue by fccoelho** [google/latexify_py#104]
I am getting the error below for the following function:
```python
@latexify.function(use_math_symbols=True)
def model(t: float, y: list, params: dict) -> list:
        """
        SIR Model.
        :param t: time step
        :param y: state of the model at time t
        :param params: parameter dictionary
        :return:
        """
        S, I, R = y
        beta, gamma, N = params['beta'], params['gamma'], params['N']
        return [
            -beta * S * I / N,
            beta * S * I / N - gamma * I,
            gamma * I
        ]
```

```python
LatexifyNotSupportedError: Codegen supports only Assign nodes in multiline functions, but got: Expr
```

If I remove the docstring, it works as expected

**Comment by soosub** [comment]
I encountered the same issue and removing the docstring resolved it. Thanks for raising the issue @fccoelho! Indeed, it would be nice to keep docstrings *and* latexify functionality all at once😌

PS cheers for maintaining this package @odashi et al., it's awesome!

**Comment by Casper-Guo** [comment]
cc:@lakeblair 

Maybe you can investigate this issue

**Comment by LakeBlair** [comment]
@odashi Hi! I can take a look at this issue and see if I can find a solution to it.

**Comment by odashi** [comment]
Hi @LakeBlair , feel free to propose a pull request for this issue.

**Comment by odashi** [comment]
Resolved by #126.
